pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -165,6 +165,19 @@ self: super: {
     meta.homepage = "https://github.com/sblumentritt/bitbake.vim/";
   };
 
+  # The GitHub repository returns 404, which breaks the update script
+  vim-pony = buildVimPluginFrom2Nix {
```

Comment:
@figsoda  shouldn't we print a warning too to mention it has gone out of fashion ?

Reply:
Sounds like a good idea, how could I add one?

Reply:
something like `vim-pony = lib.warn "The source repository doesnt exist anymore" buildVimPluginFrom2Nix { `

Reply:
That's probably better in the long run. My intention is to have a non-controversial 0-rebuilds solution that we can look back on in the future, to either deprecate/remove or maybe add back if there is a fork or if the repo was restored.